src/dataServices/dynamoDbParamBuilder.test.ts

```diff
@@ -445,3 +445,32 @@ describe('buildUpdateExportRequestJobStatus', () => {
         expect(actualParam).toEqual(clonedExpectedParam);
     });
 });
+
+describe('buildGetActiveSubscriptions', () => {
+    const expectedParam = {
+        TableName: '',
+        IndexName: 'activeSubscriptions',
+        KeyConditionExpression: '_subscriptionStatus = :active',
+        ExpressionAttributeValues: {
+            ':active': { S: 'active' },
+        },
+        ExclusiveStartKey: {},
+    };
+    test('Param without tenantId', () => {
+        const actualParam = DynamoDbParamBuilder.buildGetActiveSubscriptions();
+        expect(actualParam).toEqual(expectedParam);
+    });
+
+    test('tenantId present', () => {
+        const actualParam = DynamoDbParamBuilder.buildGetActiveSubscriptions('tenant1');
+
+        const clonedExpectedParam: any = cloneDeep(expectedParam);
```

Comment:
nit: all this cloning makes the test harder to read. reusing `expectedParam` is probably not worth it for just 2 tests.
I'd suggest declaring the whole `expectedParam` for multiTenancy or using snapshots (`expect(actualParam).toMatchInlineSnapshot`)